tests/drop_counters/test_drop_counters.py

```diff
@@ -738,6 +769,48 @@ def test_loopback_filter(ptfadapter, duthost, setup, tx_dut_ports, pkt_fields):
     testutils.verify_no_packet_any(ptfadapter, exp_pkt, ports=setup["neighbor_sniff_ports"])
 
 
+def test_ip_pkt_with_exceeded_mtu(ptfadapter, duthost, setup, tx_dut_ports, pkt_fields, mtu_setup):
+    """
+    @summary: Verify that IP packet with exceeded MTU is dropped and L3 drop counter incremented
+    """
+    dut_iface, ptf_tx_port_id, dst_mac, src_mac = get_test_ports_info(ptfadapter, duthost, setup, tx_dut_ports)
+    tmp_port_mtu = 1500
+
+    log_pkt_params(dut_iface, dst_mac, src_mac, pkt_fields["ipv4_dst"], pkt_fields["ipv4_src"])
+    # Set temporal MTU. This will be restored by 'mtu_setup' fixture
+    mtu_setup.set_mtu(tmp_port_mtu, tx_dut_ports[dut_iface])
+
+    pkt = testutils.simple_tcp_packet(
+        pktlen=9100,
+        eth_dst=dst_mac, # DUT port
+        eth_src=src_mac, # PTF port
+        ip_src=pkt_fields["ipv4_src"], # PTF source
+        ip_dst=pkt_fields["ipv4_dst"], # VM IP address
+        tcp_sport=pkt_fields["tcp_sport"],
+        tcp_dport=pkt_fields["tcp_dport"]
+        )
+
+    send_packets(pkt, duthost, ptfadapter, ptf_tx_port_id)
+
+    # Verify L3 drop counter incremented on specific interface
+    l3_drops = get_pkt_drops(duthost, "portstat -j")[dut_iface][L3_DISCARD_KEY]
+    l3_drops = int("".join(l3_drops.split(",")))
```

Comment:
Suggest to wrap this conversion into a function and name it according to the purpose of this conversion to make the code more readable.

Reply:
Fixed